metadata.json

```diff
@@ -23,24 +23,25 @@
   ],
   "description": "Using a hiera-based roles and profiles model, do simple OpenStack deployments.",
   "dependencies": [
-    { "name": "puppetlabs/keystone", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/swift", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/cinder", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/glance", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/neutron", 	"version_requirement": ">=4.2.0 <5.0.0" },
-    { "name": "puppetlabs/nova", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/heat", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/ceilometer", 	"version_requirement": ">=4.0.0 <5.0.0" },
-    { "name": "puppetlabs/horizon", 	"version_requirement": ">=4.0.0 <5.0.0" },
+    { "name": "stackforge/ceilometer", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/cinder", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/glance", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/heat", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/horizon", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/keystone", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/neutron", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/nova", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/swift", "version_requirement": ">=5.0.0 <6.0.0" },
+    { "name": "stackforge/tempest", "version_requirement": ">=5.0.0 <6.0.0" },
 
-    { "name": "puppetlabs/ntp", 	"version_requirement": ">=3.0.0 <4.0.0" },
-    { "name": "puppetlabs/firewall", 	"version_requirement": ">=1.0.0 <2.0.0" },
-    { "name": "puppetlabs/vswitch", 	"version_requirement": ">=0.2.0 <1.0.0" },
-    { "name": "puppetlabs/mongodb", 	"version_requirement": ">=0.6.0 <1.0.0" },
-    { "name": "puppetlabs/mysql", 	"version_requirement": ">=2.2.0 <3.0.0" },
-    { "name": "puppetlabs/rabbitmq", 	"version_requirement": ">=3.0.0 <4.0.0" },
-    { "name": "puppetlabs/apache", 	"version_requirement": ">=1.0.0 <1.2.0" },
+    { "name": "puppetlabs/apache", "version_requirement": ">=1.2.0 <2.0.0" },
+    { "name": "puppetlabs/firewall", "version_requirement": ">=1.0.0 <2.0.0" },
```

Comment:
The stackforge modules us this format so I'd like to keep it for consistency with them.